### ISSUE TITLE
Fix strange output after application start

### ISSUE THREAD

**Issue by ValentiWorkLearning** [Stivius/MeteoMobileApp#18]
After applciation start the debug output is filled by warn messages about unreacheable/ undefined values of properties.
Maybe this can be fixed via adding correct OpenSSL version.
![image](https://user-images.githubusercontent.com/25596072/56976497-c57d8d80-6b7b-11e9-8754-58fa44adecd4.png)

**Comment by Stivius** [comment]
No, it's because we are not retrieving forecast for the current geolocation.
